bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/ChannelTypeUtils.java

```diff
@@ -39,13 +40,20 @@
 @NonNullByDefault
 public class ChannelTypeUtils {
 
+    private static @Nullable TimeZoneProvider timeZoneProvider;
```

Comment:
Not sure I'm comfortable with adding static variables if you don't have to. But I can see why you would want to.
@J-N-K What is your opinion on this?

Reply:
I can avoid it but I should then pass TimeZoneProvier as parameter to a lot of handler classes.
If you prefer, I can do it.
Note that the astro binding uses this way with the global variable.